ramalama/common.py

```diff
@@ -243,3 +243,12 @@ def get_env_vars():
     #     env_vars[gpu_type] = str(gpu_num)
 
     return env_vars
+
+
+def rm_until_substring(model, substring):
```

Comment:
This should be a function in model.py part of the model class.

Reply:
+1 for the method.